ydb/core/protos/auth.proto

```diff
@@ -122,3 +123,13 @@ message TLdapAuthentication {
     optional string Scheme = 11 [default = "ldap"];
     optional TExtendedSettings ExtendedSettings = 12;
 }
+
+message TPasswordCheckerParameters {
+    optional uint32 MinimumLength = 1 [default = 0];
+    optional uint32 MaximumLength = 2 [default = 4294967295];
+    optional bool RestrictLower = 3;
+    optional bool RestrictUpper = 4;
+    optional bool RestrictNumbers = 5;
+    optional bool RestrictSpecial = 6;
+    optional string SpecialChars = 7 [default = "!@#$%^&*()_+{}|<>?="];
```

Comment:
Take these names from postgres. Perhaps using names as in code wil be better or otherwise.